src/plus/webviews/graph/graphWebview.ts

```diff
@@ -105,8 +105,7 @@ import type {
 	UpdateColumnsParams,
 	UpdateRefsVisibilityParams,
 	UpdateSelectedRepositoryParams,
-	UpdateSelectionParams,
-} from './protocol';
+	UpdateSelectionParams} from './protocol';
```

Comment:
Your client may have auto-formatted this, but now it's not consistent with the other import below it and should be reverted.

Reply:
Done in commit: `Fixed code formatting issues`.